mujinvisioncontrollerclient/visioncontrollerclient.py

```diff
@@ -167,12 +171,22 @@ def _WaitForResponse(self, recvjson=True, timeout=None, command=None):
         return self._ProcessResponse(response, command=command, recvjson=recvjson)
 
     def IsWaitingResponse(self):
+        # type: () -> bool
         """Returns whether the client is waiting for response on the command socket, and caller should call WaitForResponse().
         """
         return self._commandsocket.IsWaitingReply()
 
     def _SendConfiguration(self, configuration, fireandforget=False, timeout=2.0, checkpreempt=True, recvjson=True):
         # type: (Dict, bool, float, bool, bool) -> Optional[Dict]
+        """Sends a configuration command.
+
+        Args:
+            configuration (dict):
```

Comment:
Can we add a description here?

Reply:
Done. 

PS: Also added a docstring to the ExecuteCommand so it is matches with SendConfiguration.